.expeditor/config.yml

```diff
@@ -8,10 +8,15 @@ rubygems:
   - ohai
 
 github:
+  # Cleanup merged PR branches
+  delete_branch_on_merge: true
   # The tag format to use (e.g. v1.0.0)
   version_tag_format: "v{{version}}"
   # The Github Team primarily responsible for handling incoming Pull Requests.
   maintainer_group: chef/client-maintainers
+  # allow bumping the minor release via label
+  minor_bump_labels:
+    - "Expeditor: Bump Minor Version"
```

Comment:
Please use `Version: Bump Minor` so it matches idioms across other Chef projects.

Reply:
Is there a reason we don't align this all under Expeditor labels? Version: Bump Minor seems to break the mold.